app/src/main/kotlin/cz/covid19cz/app/ui/confirm/ConfirmationFragment.kt

```diff
@@ -17,25 +18,30 @@ import kotlinx.android.synthetic.main.fragment_confirmation.*
 
 abstract class ConfirmationFragment : BaseFragment<FragmentHelpBinding, ConfirmationVM>(R.layout.fragment_confirmation, ConfirmationVM::class) {
 
-    abstract val description: String
-    abstract val buttonTextRes: Int
-    abstract fun confirmedClicked()
-    abstract fun doWhenFinished()
+    abstract val confirmDescription: String
+    abstract val successShortText: String
```

Comment:
Success stuff should be in a different (abstract) SuccessFragment, not part of ConfirmationFragment